<launch>
	<!-- declare arg to be passed in -->
	<arg name="machine" default="left-cortex" />
	<arg name="user" default="strands" />
	<arg name="behaviour" default="blah" />

	<machine name="$(arg machine)" address="$(arg machine)" env-loader="/opt/strands/strands_catkin_ws/devel/env.sh" user="$(arg user)" default="true"/>

	<node pkg="qr_read_and_tweet" name="data_matrix_node" type="data_matrix_node" output="screen"/>
	<node pkg="qr_read_and_tweet" name="image_branding" type="image_branding.py" output="screen" />
	<node pkg="qr_read_and_tweet" name="read_and_tweet" type="read_and_tweet.py" output="screen" args="$(arg behaviour)"/>
</launch>
